pup-0003.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Cherry-Picking Commits
+author: David Davis
+created: 01-Apr-2017
+status:
```

Comment:
+1 to this. Since it lives as a PR for a long time I think putting 'Accepted' here will set the content up ready to be merged. It's up to you though @daviddavis